buildSrc/build.gradle

```diff
@@ -8,13 +8,16 @@ repositories {
     gradlePluginPortal()
 }
 
+java.sourceCompatibility = JavaVersion.VERSION_17
```

Comment:
here or in java-conventions? does it even matter where it is?

Reply:
Have no idea... 
moved 
```
group 'no.unit.nva'
version '1.0-JDK17'
java.sourceCompatibility = JavaVersion.VERSION_17
java.targetCompatibility = JavaVersion.VERSION_17
```
to `buildSrc/build.gradle`